managed/ui/src/components/schedules/Schedules.js

```diff
@@ -75,6 +75,19 @@ class ScheduleDisplayItem extends Component {
       !schedule.taskParams.timeBeforeDelete || schedule.taskParams.timeBeforeDelete === 0
         ? 'Unlimited'
         : moment.duration(schedule.taskParams.timeBeforeDelete).humanize();
+    
+    let backupType = '';
+    switch (schedule?.taskParams?.backupType) {
```

Comment:
We can destructure it. Initialize a variable and then put it inside the switch.

Reply:
The structure is not deeply nested so, I am leaving it for now